sdk/core/Azure.Core/perf/CustomEventSource.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace Azure.Core.Perf;
+
+/// <summary>
+/// Custom event source for Azure Core logging.
+/// </summary>
+public class CustomEventSource : EventSource
```

Comment:
It looks like this used to inherit from AzureEventSource and now it's EventSource, was this intentional to test something specific?